lib/Constants.vala

```diff
@@ -30,5 +30,7 @@ namespace Gala {
         SNAP = 250,
         // Duration of the workspace switch animation
         WORKSPACE_SWITCH = 300,
+        // Style switch animation
+        STYLE_SWITCH = 200
```

Comment:
200 would feel nice and snappy, but it seems to not be slow enough to mask all the GTK transitions. Something like 500 seems to mask them better.

```suggestion
        STYLE_SWITCH = 500
```

Reply:
Oh, nevermind, ignore me. 200 works fine if we increase the delay below slightly.